scripts/eval/yamls/model_gauntlet.yaml

```diff
@@ -0,0 +1,130 @@
+model_gauntlet:
+  weighting: EQUAL
+  subtract_random_baseline: true
+  rescale_accuracy: true
+  tasks:
+  - name: world_knowledge
+    benchmarks:
+    - name: jeopardy
+      num_fewshot: 10
+      random_baseline: 0
+    # - name: triviaqa # not used in model gauntlet v0
+    #   num_fewshot: 0
+    #   scorecard:
+    #   random_baseline: 0
```

Comment:
In MODEL_GAUNTLET.md should we mention why we exclude triviaqa and coqa?
having them here, then commenting them out will leave ppl wondering
alternatively, remove them from here
alternatively, ignore me... lol.